### ISSUE TITLE
Jax implementation

### ISSUE THREAD

**Issue by boris-kuz** [csteinmetz1/pyloudnorm#59]
Hi! 
As part of a larger project, I've implemented a fully differentiable version of this library in Jax, so if anyone wants to use this as part of a ML pipeline (for example as a part of a loss function), they can.
Repo can be found here: https://github.com/boris-kuz/jaxloudnorm
It probably doesn't make a lot of sense to merge this back, seeing as I've had to modify `integrated_loudness` quite a bit in order for it to work with Jax's `jit` engine and support `vmap`, but maybe it would make sense to have a section in the Readmes linking to tensor implementations of this library? 
There's another one for PyTorch here: https://github.com/descriptinc/audiotools/blob/master/audiotools/core/loudness.py, although this is using FIR approximations of the perceptual filters, whereas mine is using an IIR, which is quite fast thanks to the `jax.lax.scan` primitive.
Let me know your thoughts!

**Comment by csteinmetz1** [comment]
Hey @boris-kuz, this is super cool! I think adding a reference to your implementation as well as audiotools to the README is a good idea. What do you think? Go ahead and open a PR for that if you are interested. Happy to merge. Also, I will spread the word about jaxloudnorm!

**Comment by boris-kuz** [comment]
Thanks for the shoutout! :) I've updated the readme and submitted a PR.
